Lib/fontTools/ttLib/ttFont.py

```diff
@@ -1,24 +1,130 @@
+from __future__ import annotations
+
+import logging
+import os
+import traceback
+from io import BytesIO, StringIO, UnsupportedOperation
+from typing import TYPE_CHECKING, TypedDict, TypeVar, overload
+
 from fontTools.config import Config
 from fontTools.misc import xmlWriter
 from fontTools.misc.configTools import AbstractConfig
-from fontTools.misc.textTools import Tag, byteord, tostr
 from fontTools.misc.loggingTools import deprecateArgument
+from fontTools.misc.textTools import Tag, byteord, tostr
 from fontTools.ttLib import TTLibError
+from fontTools.ttLib.sfnt import SFNTReader, SFNTWriter
 from fontTools.ttLib.ttGlyphSet import (
-    _TTGlyph,
+    _TTGlyph,  # noqa: F401
+    _TTGlyphSet,
     _TTGlyphSetCFF,
     _TTGlyphSetGlyf,
     _TTGlyphSetVARC,
 )
-from fontTools.ttLib.sfnt import SFNTReader, SFNTWriter
-from io import BytesIO, StringIO, UnsupportedOperation
-import os
-import logging
-import traceback
+
+if TYPE_CHECKING:
+    from collections.abc import Mapping, MutableMapping
+    from types import ModuleType, TracebackType
+    from typing import Any, BinaryIO, Literal, Sequence, TextIO
+
+    from typing_extensions import Self, Unpack
+
+    from fontTools.ttLib.tables import (
```

Comment:
Can this import be move to `fontTools/ttLib/tables/__init__.py` and table modules accessed bellow as `tables.B_A_S_E_` etc. to avoid this very long import here? Adding it there might also increase the chance of keeping the list up to date.